Core/HLE/sceNet.h

```diff
@@ -186,11 +194,43 @@ struct ApctlHandler {
 	u32 argument;
 };
 
+struct ApctlArgs {
+	u32_le data[5]; // OldState, NewState, Event, Error, ArgsAddr
+};
+
 class PointerWrap;
 
+class AfterApctlMipsCall : public PSPAction {
+public:
+	AfterApctlMipsCall() {}
+	static PSPAction* Create() { return new AfterApctlMipsCall(); }
```

Comment:
Does using `static` here could cause issue? i wonder if this have something to do with the weird things happening after loading from old save state.

Reply:
Yeah static here is not an issue, it's intentional.